client/state-db/src/noncanonical.rs

```diff
@@ -870,13 +874,10 @@ mod tests {
 	fn complex_tree() {
 		let mut db = make_db(&[]);
 
-		// - 1 - 1_1 - 1_1_1
-		//     \ 1_2 - 1_2_1
-		//           \ 1_2_2
-		//           \ 1_2_3
 		//
-		// - 2 - 2_1 - 2_1_1
-		//     \ 2_2
+		// - 1 - 1_1 - 1_1_1 \ 1_2 - 1_2_1 \ 1_2_2 \ 1_2_3
+		//
+		// - 2 - 2_1 - 2_1_1 \ 2_2
```

Comment:
oh dear

Reply:
did `#[rustfmt::skip]` for this one.

Reply:
Could you re-do it please? Seems to be broken.